### PR TITLE
Main view skeleton

### PR DESCRIPTION
**Pull request by kennyczadzeck**
Add main layout view skelton to hold primary Angular directives.

### CODE REVIEW

**Comment by krish-andres** [comment]
Tested, looks good

**Comment by mmomen** [comment]
Tested, looks good.

File: views/layout.ejs [hunk 1]
```diff
@@ -39,8 +39,13 @@
   </head>
 
   <body>
-    
-    <div ng-view=""></div>
+
+      <div class="col-lg-3">
```
Comment (Dudemullet) [inline views/layout.ejs]: In order to use Bootstraps grid system, your columns should be inside a `.row`. [reference](http://getbootstrap.com/2.3.2/scaffolding.html#gridSystem). Please include these columns inside a `.row` wrapper

**Comment by Dudemullet** [comment]
@kennyczadzeck please rebase and squash so I can merge

**Comment by Dudemullet** [comment]
:ship:

**Pull request merged**